src/imp/span_context_imp.js

```diff
@@ -0,0 +1,40 @@
+export default class SpanContextImp {
+
+    // ---------------------------------------------------------------------- //
+    // OpenTracing Implementation
+    // ---------------------------------------------------------------------- //
+
+    setBaggageItem(key, value) {
+        this._baggage[key] = value;
+    }
+
+    getBaggageItem(key) {
+        return this._baggage[key];
+    }
+
+    // ---------------------------------------------------------------------- //
+    // LightStep Extensions
+    // ---------------------------------------------------------------------- //
+
+    // This is part of the formal OT API in Go; and will likely be supported
+    // across platforms.
+    //
+    // https://github.com/opentracing/opentracing.github.io/issues/103
+    foreachBaggageItem(f) {
```

Comment:
https://gitter.im/opentracing/public?at=57729f81a0c12d110fae397e

I.e., this was intentional, though I don't feel strongly about it.